viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    # if any of the values in our inputs are falsy 
+    result = None
+    
+
+    if not title or not genre or not rating:
+        return None 
+    else:
+        new_movie = {}
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating

```

Comment:
Instead of assigning a new key-value pair every line, we can do it in one go:

```suggestion
        new_movie = {
             "title": title,
             "genre": genre,
             "rating": rating
         }
```